### ISSUE TITLE
GloFAS version 4

### ISSUE THREAD

**Issue by turnerm** [OCHA-DAP/ocha-anticipy#182]
Version 4 of the GloFAS model (reanalysis only so far) has been released, we should make it possible to download it

**Comment by aonomike** [comment]
Was this made available for download?

**Comment by turnerm** [comment]
@aonomike yes version 4 should be supported, although I haven't checked it in a few months so things may have changed. And the reporting points have not been updated for version 4 unfortunately.
